src/store/messages/saga.ts

```diff
@@ -90,6 +92,45 @@ const _isActive = (channelId) => (state) => {
 
 const FETCH_CHAT_CHANNEL_INTERVAL = 60000;
 
+function* getZeroUsersMap() {
+  const users = yield select((state) => state.normalized.users);
+  const zeroUsersMap: { [matrixId: string]: User } = {};
+  for (const user of Object.values(users)) {
+    zeroUsersMap[(user as User).matrixId] = user as User;
+  }
+  // map current user as well
+  const currentUser = yield select(currentUserSelector());
+  zeroUsersMap[currentUser.matrixId] = {
+    userId: currentUser.id,
+    profileId: currentUser.profileSummary.id,
+    firstName: currentUser.profileSummary.firstName,
+    lastName: currentUser.profileSummary.lastName,
+    profileImage: currentUser.profileSummary.profileImage,
+  } as User;
+
+  return zeroUsersMap;
+}
+
+export function* mapMessageSenders(messages) {
+  if (!featureFlags.enableMatrix) {
+    return;
+  }
+
+  const zeroUsersMap = yield call(getZeroUsersMap);
+  messages.forEach((message) => {
+    const zeroUser = zeroUsersMap[message.sender.userId];
```

Comment:
Does `sender` have the `matrixId`? If so, I think it makes sense to use that here rather than the `userId`. I know they're currently the same but `matrixId` is more clear.

Reply:
its doesn't :(

<img width="465" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/422ad84c-117a-471e-b609-9f84324876b9">

I will add a comment here